### PR TITLE
Disable network fees for custom networks.

### PR DESCRIPTION
**Pull request by 0xDaedalus**
Closes https://github.com/tahowallet/extension/issues/3189

Screen Shot 2023-03-28 at 11 08 09 AM

### To Test
- Add the Evmos chain to the wallet
- Attempt to send EVMOS to an address
- Notice that gas settings are greyed out with the correct tooltip.

Latest build: [extension-builds-3203](https://github.com/tahowallet/extension/suites/12040401263/artifacts/633008979) (as of Wed, 05 Apr 2023 08:40:51 GMT).

### CODE REVIEW

**Review by jagodarybacka — DISMISSED** [review]
Shouldn't it be the same styles as regular network settings button but without the border? Font looks smaller and the $ value is not white 🤔

**Review by hyphenized — DISMISSED** [review]
We should also remove the $ value as it is not based on the network asset price

File: ui/components/NetworkFees/FeeSettingsButton.tsx [hunk 1]
```diff
@@ -8,6 +14,37 @@ interface FeeSettingsButtonProps {
...
+      <div>
+        <SharedTooltip
+          width={175}
+          height={30}
+          type="dark"
+          horizontalShift={90}
+          horizontalPosition="center"
+          verticalPosition="bottom"
+          IconComponent={() => (
+            <div className="disabled_settings">
```
Comment (jagodarybacka) [inline ui/components/NetworkFees/FeeSettingsButton.tsx:34]: Styles in `styled-jsx` are not going through the components to the child components so `.disabled_settings` styles are not applied here 😞

**Review by kkosiorowska — APPROVED** [review]
The comments were implemented. Let's merge it.✅

Screenshot 2023-04-05 at 10 36 28

**Pull request merged**